go/vt/vttablet/tabletmanager/vreplication/vreplicator.go

```diff
@@ -580,10 +580,18 @@ func (vr *vreplicator) throttlerAppName() string {
 	return throttlerapp.Concatenate(names...)
 }
 
+// updateTimeThrottled updates the time_throttled field in the _vt.vreplication record
+// with a rate limit so that it's only saved in the database at most once per
+// throttleUpdatesRateLimiter.tickerTime.
+// It also increments the throttled count in the stats to keep track of how many
+// times a VReplication workflow, and the specific sub-component, is throttled by the
+// tablet throttler over time.
 func (vr *vreplicator) updateTimeThrottled(appThrottled throttlerapp.Name) error {
+	at := appThrottled.String()
```

Comment:
English question: "at" implies a _time_, but here "at" is the app name? Perhaps a different variable name?

Reply:
I'll move it to appName